tensor2tensor/trax/layers/initializers.py

```diff
@@ -30,21 +41,70 @@ def Init(shape, rng):
   return Init
 
 
-def GlorotNormalInitializer(out_dim=0, in_dim=1, scale=onp.sqrt(2)):
-  """An initializer function for random Glorot-scaled coefficients."""
+def RandomUniformInitializer(lim=1.0):
+  """An initializer function for random uniform coefficients."""
   def Init(shape, rng):
-    fan_in, fan_out = shape[in_dim], shape[out_dim]
-    size = onp.prod(onp.delete(shape, [in_dim, out_dim]))
-    std = scale / backend.numpy.sqrt((fan_in + fan_out) / 2. * size)
-    return (std * backend.random.normal(rng, shape)).astype('float32')
+    return (backend.random.uniform(rng, shape, -lim, lim)).astype('float32')
   return Init
 
 
-def GlorotUniformInitializer(out_dim=0, in_dim=1):
```

Comment:
Shouldn't the default out_dim and in_dim be -1 and -2 as above in all initializers?

Reply:
Yes, you're correct again, I used what was written in the main branch without thinking. It is fixed now